pkg/router/template/types.go

```diff
@@ -74,6 +74,12 @@ type ServiceAliasConfig struct {
 
 	// ActiveEndpoints is a count of the route endpoints that are part of a service unit with a non-zero weight
 	ActiveEndpoints int
+
+	// For route based setting of http response headers
```

Comment:
nit: upper case all the `http` references

Reply:
:+1:, and please also put the field name as the first word in the comment, and ensure you end the sentence with a period:
```suggestion
	// HTTPResponseHeaders has route-specific custom HTTP response headers.

Reply:
done.